languages/scala-2/template/build.sbt

```diff
@@ -0,0 +1,17 @@
+name := "algorithm"
+organization := "algorithmia"
+version := "0.1"
+scalaVersion := "2.13.1"
+
+val repoUrl = System.getProperty("repo.url", "http://git.algorithmia.com")

```

Comment:
Is this used by anything?

Reply:
I just copied that from the existing scala build.sbt, if not necessary I'll remove